script.js

```diff
@@ -0,0 +1,212 @@
+const filteredStatus = (arr) => {
+    let toDoStatus = arr.filter( el => el.status === 'to_do');
+    let inProgressStatus = arr.filter( el => el.status === 'in_progress');
+    let doneStatus = arr.filter( el => el.status === 'done');
+
+    return { toDoStatus, inProgressStatus, doneStatus}
+};
+
+const newTaskBtn = document.getElementById('new_task_btn');
+const newTaskInput = document.getElementById('new_task_text');
+const newTaskForm = document.getElementById('new_task_form');
+
+const selectOption = document.getElementById('select_progress');
+
+const toDoContainer = document.getElementById('to_do');
+const inProgressContainer = document.getElementById('in_progress');
+const doneContainer = document.getElementById('done');
+
+const sortBtnToDo = document.getElementById('sort_btn_to_do');
+const sortBtnInProgress = document.getElementById('sort_btn_in_progress');
+const sortBtnDone = document.getElementById('sort_btn_done');
+
+const editWrapForm = document.querySelector('.hidden.edit_form_wrap')
+const editTaskForm = document.getElementById('edit_task_form');
+const editTaskInput = document.getElementById('edit_form_text');
+const editTaskBtn = document.getElementById('edit_task_btn');
+const idInput = document.getElementById('task_id');
+const closeTaskBtn = document.getElementById('close_form_btn');
+
+const editSelectOption = document.getElementById('edit_select_progress');
+
+
+
+newTaskInput.addEventListener('click', () => {
+    newTaskInput.value = '';
+});
+
+newTaskInput.addEventListener('blur', () => {
+    if (newTaskInput.value === '') {
+        newTaskInput.value = 'ADD A TASK';
+    }
+});
+
+//Refresh container with tasks
+const refreshView = (taskArray) => {
```

Comment:
this function can be independent of any params. Task array is always db from localstorage right? So it can get data from local storage by its own.